webapp/graphql/Fragments.js

```diff
@@ -62,6 +62,18 @@ export const postFragment = gql`
   }
 `
 
+export const groupFragment = gql`
+  fragment group on Group {
+    id
+    groupName: name
+    slug
+    disabled
+    deleted
+    about
+    groupType
+  }
+`
```

Comment:
```suggestion
export const groupFragment = gql`
  fragment group on Group {
    id
    groupName: name
    slug
    disabled
    deleted
    about
    description
    descriptionExcerpt
    groupType
    actionRadius
    categories {
      id
      slug
      name
      icon
    }
    locationName
    location {
      name: name${lang}
    }
    myRole
  }
`
```

Some properties are missing here …
And therefore `i18n`.

And why is `groupName: name` necessary?